Book (#262)
epiphany-platform/CommonDataLayer: docs/architecture/ingestion.md

# Ingestion Layer

> Services in this layer are responsible for accepting generic messages from external systems via `Kafka`, validating them and sorting to correct repository.  

Reviewer: ```suggestion
Services in this layer are responsible for accepting generic messages from external systems via a message queue, validating them and and forwarding the message to correct repository.  
```
